docs/guides/guides/command-line.mdx

```diff
@@ -122,6 +122,8 @@ cypress run [options]
 | `--group`                      | [Group recorded tests together under a single run](#cypress-run-group-lt-name-gt)                                                                                                             |
 | `--headed`                     | [Displays the browser instead of running headlessly](#cypress-run-headed)                                                                                                                     |
 | `--headless`                   | [Hide the browser instead of running headed (default during `cypress run`)](#cypress-run-headless)                                                                                            |
+| `--runner-ui`                  | [Display the command log, application under test url bar, viewport preview, and browser dropdown while recording Test Replay (can negatively impact performance)](#cypress-run-runner-ui)     |
```

Comment:
@jennifer-shehane I added in these flags. let me know if anything needs to be changed.

Reply:
@jaffrepaul They're already documented in this seperate PR: https://github.com/cypress-io/cypress-documentation/pull/5424

Reply:
cool will drop this commit